### PR TITLE
[Universalbots][Createdconceptualdocument][1149654]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Created conceptual document for Universal Bots in Teams. But still awaiting inputs for queries with Ojasvi Choudhary regarding footnotes mentioned in the word document. Please review this pull request.

### CODE REVIEW

File: msteams-platform/TOC.yml [hunk 1]
```diff
@@ -267,6 +267,8 @@
         href: bots/calls-and-meetings/debugging-local-testing-calling-meeting-bots.md
       - name: Application-hosted media
         href: bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md
+    - name: Universal Bots in Microsoft Teams
+      href: bots/Universal-Bots-in-Teams.md
```
Comment (AkJo) [inline msteams-platform/TOC.yml:271]: Are not all filenames supposed to be all lowercase?

**Comment by laujan** [comment]
@surbhigupta12; @v-rodriguezd 

 
image

File: msteams-platform/bots/universal-bots-in-teams.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
+---
+title: Universal Bots in Microsoft Teams
+PM:  Ojasvi Choudhary (transitioning from Bill Bliss)
+description: Understand Universal Bots in Teams
+ms.topic: conceptual
+---
+
+# Universal Bots in Microsoft Teams
```
Comment (ojasvichoudhary) [inline msteams-platform/bots/universal-bots-in-teams.md:8]: Can we rename this to - Universal Actions for Adaptive cards in Microsoft Teams

**Comment by surbhigupta12** [comment]
Closing this PR as it is addressed in PR #2769

**Pull request closed without merging**